pom.xml

```diff
@@ -529,6 +523,13 @@
           <debug>true</debug>
           <debuglevel>lines,vars,source</debuglevel>
           <compilerArgument>-proc:none</compilerArgument>
+          <annotationProcessorPaths>
+            <annotationProcessorPath>
+              <groupId>org.hibernate</groupId>
+              <artifactId>hibernate-jpamodelgen</artifactId>
+              <version>5.6.12.Final</version>
+            </annotationProcessorPath>
+          </annotationProcessorPaths>
```

Comment:
Can you give some context around this change? How does it relate to RHEL9 / rpm build?

Reply:
Maven build was throwing a duplicate dependency error. Our current setup was generating compile dependencies for annotation processors so I took a look what is a Maven preferred way of defining annotation processors.